src/parsers/i3d/unpackGeometry/main.ts

```diff
@@ -151,10 +135,13 @@ function findOrCreateDestinationGroup(
   if (destinationGroup !== undefined) {
     return destinationGroup;
   } else {
+    let primitivesPerSectorAndChildren = 0;
```

Comment:
If this is a count, use `count` or `num`, or `numberOf` in the variable name maybe?

Reply:
Fixed